docs/src/tutorials/implementing.md

```diff
@@ -77,7 +77,7 @@ easiest way to do this is to copy an existing solver. Good examples to follow
 are the [COIN-OR solvers](https://github.com/JuliaPackaging/Yggdrasil/tree/master/C/Coin-OR).
 
 !!! warning
-    Building the solver via Yggdrasil is non-trivial. please ask the
+    Building the solver via Yggdrasil is non-trivial. Please ask the
     [Developer chatroom](https://gitter.im/JuliaOpt/JuMP-dev) for help.
```

Comment:
I wonder if we should link to https://matbesancon.github.io/post/2019-11-04-binary-julia/